VultisigApp/VultisigApp/Model/TransactionMemo/TransactionMemoBondMayaChain.swift

```diff
@@ -40,30 +41,45 @@ class TransactionMemoBondMayaChain: TransactionMemoAddressable, ObservableObject
             }
         }
     }
-    
+
     private var cancellables = Set<AnyCancellable>()
-    
-    required init(assets: [String]) {
+
+    let id = UUID()
```

Comment:
this one doesn't used any where , could you please remove it

Reply:
removed